docs/products/kafka/howto/configure-log-cleaner.rst

```diff
@@ -0,0 +1,30 @@
+Configure log cleaner for compaction
+=====================================
+
+By default the log cleaner is enabled, but the log compaction is disabled. To enable log compaction, follow these steps:
+
+#. Log into the `Aiven web console <https://console.aiven.io/>`_ and select your Aiven for Apache Kafka service.
+#. Scroll down to *Advanced configuration* and click on **Add configuration option**.
+#. Find ``log.cleanup.policy`` in the list and select it.
+#. Set the value to ``compact``.
+
+This change will affect all topics in the cluster that do not have a configuration override in place.
+
+Control the frequency and delay
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before the cleaning begins, the cleaner thread will inspect the logs to find those with highest **dirty ratio** which is calculated as the ratio of the number of bytes in the head vs the total number of bytes in the log (tail + head). The default ratio value is 50%, but you can change it by change value of the property ``kafka.log_cleaner_min_cleanable_ratio`` from the *Advanced configuration* list.
```

Comment:
Why is this ratio used? Adding a bit more explanation telling that this ratio allows to understand how many duplicated keys we have in a topic, and those will need to be compacted?